client/src/components/PhotoCarousel.jsx

```diff
@@ -1,28 +1,58 @@
 import React, { Component } from 'react';
-import Photo from './Photo';
-import Modal from './Modal';
+import '../../dist/style.css';
+
+const imgStyle = {
+  height: '395px',
+  width: '395px',
+  maxWidth: '100%',
+  maxHeight: '395px',
+  margin: '3px',
+};
+
+const imgColStyle = {
+  height: '195px',
+  width: '195px',
+  maxWidth: '100%',
+  maxHeight: '195px',
+  margin: '3px',
+};
+
+const sectionStyle = {
+  display: 'flex',
+  justifyContent: 'center',
+  flexWrap: 'wrap',
+};
 
 export default class PhotoCarousel extends Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      images: [
-        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/aurora.jpg',
-        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/canyon.jpg',
-        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/city.jpg',
-        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/desert.jpg',
-        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/mountains.jpg',
-        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/redsky.jpg',
-        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/sandy-shores.jpg',
-        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/tree-of-life.jpg',
-        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/aurora.jpg',
-        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/city.jpg',
-        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/mountains.jpg',
-        'https://s3.us-east-2.amazonaws.com/dzuz14/thumbnails/redsky.jpg',
-      ],
+      mediaData: [],
       showModal: false,
     };
+    const { mediaData } = this.state;
+    this.element = document.createElement('img');
+    this.element.setAttribute('src', mediaData[2]);
+    this.element.setAttribute('height', '195');
+    this.element.setAttribute('width', '195');
+    this.element.setAttribute('style', 'margin: 3px');
+  }
+
+  // document.getElementById('smallMedia').appendChild(this.element);
+
+  componentDidMount() {
+    const path = window.location.pathname;
+    const id = path.split('-')[2].split('/')[0];
+    const endpoint = `http://localhost:3002/restaurant/${id}`;
```

Comment:
Reconsider your endpoint to be something like '/api/.....' to make it distinct from where you app is hosted on '/r/....'